extensions/amp-consent/0.1/consent-config.js

```diff
@@ -37,6 +38,7 @@ const ALLOWED_DEPR_CONSENTINSTANCE_ATTRS = {
 /** @const @type {!Object<string, boolean>} */
 const CONSENT_VARS_ALLOWED_LIST = {
   'CLIENT_ID': true,
+  'CONSENT_PAGE_VIEW_ID_64': true,
```

Comment:
We don't need these two any longer, since we set them from the resolved vars below.

```suggestion
```